DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/src/FakeIOPipes.hpp

```diff
@@ -245,21 +248,18 @@ class ConsumerImpl : public ProducerConsumerBaseImpl<Id, T, use_host_alloc> {
     });
 
     // if the user wanted to use board memory, copy the data back to the host
+    event dma_event;

```

Comment:
Should we initialized dma_event?  Or does it have a constructor that sets it to a reasonable default?

Reply:
It will be initialized to a reasonable default (think of it like a 'null' event). If we call `.wait()` on it, it does nothing.